src/web3/networkConfig.ts

```diff
@@ -180,11 +184,29 @@ const CELO_EXPLORER_BASE_ADDRESS_URL_MAINNET = `${CELO_EXPLORER_BASE_URL_MAINNET
 const CELO_EXPLORER_BASE_TOKEN_URL_ALFAJORES = `${CELO_EXPLORER_BASE_URL_ALFAJORES}/token/`
 const CELO_EXPLORER_BASE_TOKEN_URL_MAINNET = `${CELO_EXPLORER_BASE_URL_MAINNET}/token/`
 
-const ETH_EXPLORER_BASE_URL_SEPOLIA = 'https://sepolia.etherscan.io'
-const ETH_EXPLORER_BASE_URL_MAINNET = 'https://etherscan.io'
+const CELOSCAN_BASE_URL_ALFAJORES = 'https://alfajores.celoscan.io'
+const CELOSCAN_BASE_URL_MAINNET = 'https://celoscan.io'
+
+const CELOSCAN_BASE_TX_URL_ALFAJORES = `${CELOSCAN_BASE_URL_ALFAJORES}/tx/`
+const CELOSCAN_BASE_TX_URL_MAINNET = `${CELOSCAN_BASE_URL_MAINNET}/tx/`
+
+const CELOSCAN_BASE_ADDRESS_URL_ALFAJORES = `${CELOSCAN_BASE_URL_ALFAJORES}/address/`
+const CELOSCAN_BASE_ADDRESS_URL_MAINNET = `${CELOSCAN_BASE_URL_MAINNET}/address/`
+
+const CELOSCAN_BASE_TOKEN_URL_ALFAJORES = `${CELOSCAN_BASE_URL_ALFAJORES}/token/`
+const CELOSCAN_BASE_TOKEN_URL_MAINNET = `${CELOSCAN_BASE_URL_MAINNET}/token/`
+
+const ETHERSCAN_BASE_URL_SEPOLIA = 'https://sepolia.etherscan.io'
+const ETHERSCAN_BASE_URL_MAINNET = 'https://etherscan.io'
+
+const ETHERSCAN_BASE_TX_URL_SEPOLIA = `${ETHERSCAN_BASE_URL_SEPOLIA}/tx/`
+const ETHERSCAN_BASE_TX_URL_MAINNET = `${ETHERSCAN_BASE_URL_MAINNET}/tx/`
 
-const ETH_EXPLORER_BASE_TX_URL_SEPOLIA = `${ETH_EXPLORER_BASE_URL_SEPOLIA}/tx/`
-const ETH_EXPLORER_BASE_TX_URL_MAINNET = `${ETH_EXPLORER_BASE_URL_MAINNET}/tx/`
+const ETHERSCAN_BASE_ADDRESS_URL_SEPOLIA = `${ETHERSCAN_BASE_URL_SEPOLIA}/address/`
+const ETHERSCAN_BASE_ADDRESS_URL_MAINNET = `${ETHERSCAN_BASE_URL_MAINNET}/address/`
+
+const ETHERSCAN_BASE_TOKEN_URL_SEPOLIA = `${ETHERSCAN_BASE_URL_SEPOLIA}/token/`
+const ETHERSCAN_BASE_TOKEN_URL_MAINNET = `${ETHERSCAN_BASE_URL_MAINNET}/token/`
```

Comment:
i'm wondering if there's any benefit to having these urls (excluding the actual base url) declared here and then used in `blockExplorerUrls`, would it be more readable to create these directly inside `blockExplorerUrls`?